packages/@webex/plugin-cc/src/cc.ts

```diff
@@ -48,7 +49,9 @@ export default class ContactCenter extends WebexPlugin implements IContactCenter
         webex: this.$webex,
       });
 
-      this.services = Services.getInstance();
+      this.webSocketManager = new WebSocketManager({webex: this.$webex});
+
+      this.services = Services.getInstance(this.webSocketManager);
```

Comment:
Here we will inject the manager for reasons that will be explained in the Service class implementation.